bin/capsule-remove

```diff
@@ -84,17 +45,15 @@ LOG_FILES=(
     /var/log/foreman-installer
     /var/log/squid
 )
-
+ 
 # Setting a variable to make sure we actually grab the FQDN of the capsule incase there are other packages that have a short hostname in them.
-
+ 
 FQDN=$( hostname -f)
-
+ 
 RPMS=(
     `rpm -qa | grep puppetlabs-release`
     `rpm -qa | grep foreman-release`
     `rpm -qa | grep foreman-client`
-    `rpm -qa | grep ^foreman-installer`
-    `rpm -qa | grep ^katello-installer`
```

Comment:
@beav lines 58 and 66 are removing those with the # rpm -qa | grep ^foreman and ^katello so removing extra lines of code. Tested on a install and those greps return nothing after the script runs